expr/service_test.go

```diff
@@ -43,6 +43,42 @@ func TestServiceExprMethod(t *testing.T) {
 	}
 }
 
+func TestServiceExprError(t *testing.T) {
+	var (
+		errorFoo = &ErrorExpr{
+			Name: "foo",
+		}
+		errorBar = &ErrorExpr{
+			Name: "bar",
+		}
+	)
+	cases := map[string]struct {
+		name     string
+		expected *ErrorExpr
+	}{
+		"exist": {
+			name:     "foo",
+			expected: errorFoo,
+		},
+		"not exist": {
+			name:     "baz",
+			expected: nil,
+		},
+	}
+
+	for k, tc := range cases {
```

Comment:
Let's start using `t.Run` for new tests. This we don't need to print `k` in `t.Errorf`.
```go
for k, tc := range cases {
  t.Run(k, func(t *testing.T) {
    ...
    if actual := s.Error(tc.name); actual != tc.expected {
      t.Errorf("invalid error expression: got %#v, expected %#v", actual, tc.expected)
    }
  })
}
```

Reply:
https://github.com/goadesign/goa/pull/2024/commits/e21a2308251d4044c2c6bd6106ed7a61992958ae